examples/scripts/script_reslc.py

```diff
@@ -0,0 +1,143 @@
+"""Example script of calculating SLC from interferograms.
```

Comment:
Reformulate to
Example script for calculating SLCs from interferograms.

Reply:
```suggestion
"""Example script for calculating SLCs from interferograms.
```